config/initializers/unify_google_compute.rb

```diff
@@ -100,9 +154,16 @@ def create_disk(params)
             # rethought.
             size_gb: 10,
             zone_name: params[:zone],
-            source_image: params[:image_id],
             autoDelete: true
```

Comment:
I already answered similar question here https://github.com/dice-cyfronet/atmosphere/pull/134#discussion_r27894341 :smile:. It is required because some of the parameter is used by fog client and other are directly translated into json structure sent to google compute.